src/store/TranscriptionsStore.js

```diff
@@ -56,6 +56,14 @@ const TranscriptionsStore = types.model('TranscriptionsStore', {
     })
   },
 
+  deleteCurrentLine: function deleteCurrentLine() {
+    const index = getRoot(self).subjects.index
+    const page = self.current.text.get(`frame${index}`)
+    page.splice(self.activeTranscriptionIndex, 1)
+    self.saveTranscription()
+    self.setActiveTranscription()
+  },
```

Comment:
This function looks legit, and basic tests shows that it works fine... (e.g. `self.setActiveTranscription(nothing)` correctly resets the actively selected item after it's been deleted, which was the first thing I looked for) 

I'm wondering if there's wisdom in adding a check to make sure `(self.activeTranscriptionIndex !== null && self.activeTranscriptionIndex !== undefined)`, or `(Number.isInteger(self.activeTranscriptionIndex))` as a safety. I mean, the _UI_ makes it practically impossible for `activeTranscriptionIndex` to be undefined when the Delete modal appears, so I might just be too paranoid.

(Note: `arr.splice(undefined, 1)` works the same as `arr.splice(0, 1)`)

(Reminder: please don't use `(!!self.activeTranscriptionIndex)` since that'll trip up when the index is 0.)

Reply:
Although a user _shouldn't_ be able to delete a transcription without one selected, I think it is a smart idea to add a check here.